src/main/webapp/app/services/visualizations/clusteredhorizontalbar-chart.service.js

```diff
@@ -49,6 +49,7 @@
                     result['isFilterGrid'] = VisualizationUtils.getPropertyValue(record.properties, 'Show Filter Grid');
                     result['showSorting'] = VisualizationUtils.getPropertyValue(record.properties, 'Allow Sorting');
                     result['displayName'] = VisualizationUtils.getFieldPropertyValue(dimensions[0], 'Display name') || result['dimension'][0];
+                    result['alternateDimension'] = VisualizationUtils.getPropertyValue(record.properties, 'Alternative Dimensions');
```

Comment:
why we are keeping visualization related files inside flair-bi...we have them already in flair-visualisation project

Reply:
it's not visualization related files 
its visualization service file that calls flair-visualisation